src/object/rigid_body.rs

```diff
@@ -614,6 +622,18 @@ impl<N: Scalar> RigidBody<N> {
         self.update_inertia_tensor();
     }
 
+    /// Reference to the collision groups of this rigid body.
+    #[inline]
+    pub fn collision_groups(&self) -> &RigidBodyCollisionGroups {
+        &self.collision_groups
+    }
+
+    /// Mutable reference to the collision groups of this rigid body.
+    #[inline]
+    pub fn collision_groups_mut(&mut self) -> &mut RigidBodyCollisionGroups {
```

Comment:
I am not 100% sure but I think it is better not to return a mutable reference, but instead have something like `.set_collision_groups(&mut self, new_groups: &mut RigidBodyCollisionGroups)`. In the future, the rigid body will have to know that the groups have been tempered with in order to notify the collision detectors.